CHANGES

```diff
@@ -1,5 +1,6 @@
 Back In Time
 Version 1.5.3-dev (development of upcoming release)
+* Fix: Correct global flock fallbacks (#1834)
```

Comment:
You can also put your name as author behind that line. Using nickname or full name is up to you. Like this:

    * Fix: Correct global flock fallbacks (#1834) (Timothy Southwick)